src/test/java/core/basesyntax/service/impl/CsvFileReportWriterServiceTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.ReportRecord;
+import core.basesyntax.service.ReportWriterService;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class CsvFileReportWriterServiceTest {
```

Comment:
Please do not be in a hurry with tests.
They will be in the next task. This one doesn't require them.
For the moment we should agree on code structure